packages/react-reconciler/src/__tests__/ReactSuspenseWithNoopRenderer-test.internal.js

```diff
@@ -1842,6 +1842,80 @@ describe('ReactSuspenseWithNoopRenderer', () => {
     expect(ReactNoop.getChildren()).toEqual([span('A'), span('C'), span('B')]);
   });
 
+  it('favors showing the inner boundary for nested top level avoided boundaries', async () => {
```

Comment:
So for this, the inner suspense block should show in the parent Suspense when nesting, or is there additional detail for this test...?